wrappers/opencv/face/rs-face-cv.cpp

```diff
@@ -0,0 +1,65 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2017 Intel Corporation. All Rights Reserved.
+
+#include <librealsense2/rs.hpp> // Include RealSense Cross Platform API
+#include <opencv2/opencv.hpp>   // Include OpenCV API
+#include "../cv-helpers.hpp"
+
+int main(int argc, char * argv[]) try
+{
+    // Declare depth colorizer for pretty visualization of depth data
+    rs2::colorizer color_map;
+
+    // Declare RealSense pipeline, encapsulating the actual device and sensors
+    rs2::pipeline pipe;
+    // Start streaming with default recommended configuration
+    pipe.start();
+
+    using namespace cv;
+    const auto window_name = "Display Image";
+    namedWindow(window_name, WINDOW_AUTOSIZE);
+
+	CascadeClassifier cascade( "C:\\work\\opencv\\sources\\data\\haarcascades\\haarcascade_frontalface_default.xml" );
```

Comment:
Remove hard-coded pathes

Reply:
This file needs to be taken out of the changes -- the opencv/face/ example is not going in...